gateway-ha/src/main/java/io/trino/gateway/ha/router/TrinoQueryProperties.java

```diff
@@ -238,11 +241,37 @@ else if (statement instanceof ExecuteImmediate executeImmediate) {
         }
     }
 
-    private Map<String, String> getPreparedStatements(HttpServletRequest request)
+    private void processRequestBody(ContainerRequestContext requestContext)
```

Comment:
Maybe clarify name such as "processRequestBodyFromContext"?

Reply:
The parameter kinda tells it, so it would be redundant.